php/php.editor/src/org/netbeans/modules/php/editor/PredefinedSymbols.java

```diff
@@ -107,13 +107,57 @@ public final class PredefinedSymbols {
                 "__unserialize", // NOI18N PHP 7.4
             })));
 
+    public static final Set<String> ATTRIBUTE_NAMES = Collections.unmodifiableSet(Attributes.ATTRIBUTE_NAMES);
+    public static final Set<String> ATTRIBUTE_FQ_NAMES = Collections.unmodifiableSet(Attributes.ATTRIBUTE_FQ_NAMES);
+
     public static enum VariableKind {
         STANDARD,
         THIS,
         SELF,
         PARENT
     };
 
+    // https://www.php.net/manual/en/reserved.attributes.php
+    public static enum Attributes {
+        ATTRIBUTE("Attribute"), // NOI18N
+        ALLOW_DYNAMIC_PROPERTIES("AllowDynamicProperties"), // NOI18N
+        OVERRIDE("Override"), // NOI18N
+        RETURN_TYPE_WILL_CHANGE("ReturnTypeWillChange"), // NOI18N
+        SENSITIVE_PARAMETER("SensitiveParameter"), // NOI18N
+        ;
+
+        private final String name;
+        private final String fqName;
+        private final String asAttributeExpression;
+        private static final Set<String> ATTRIBUTE_NAMES = new HashSet<>();
+        private static final Set<String> ATTRIBUTE_FQ_NAMES = new HashSet<>();
```

Comment:
These 2 constants should be defined above the class members.

Reply:
Thanks!

Reply:
Fixed: https://github.com/apache/netbeans/compare/3959424d4024dd610ca0c912e1448a8f1a4d5762..e79431d3a47a928b681752608e74fe49743338b5